editions/tw5.com/tiddlers/macros/examples/lingo Macro (language plugin examples).tid

```diff
@@ -1,7 +1,7 @@
 created: 20150221151358000
 modified: 20150221160113000
 tags: [[lingo Macro]] [[Macro Examples]]
-title: lingo Macro (Examples)
+title: lingo (for official language plugin) Macro (Examples)
```

Comment:
IMO having 2 (paranthesis) in 1 tiddler title is not good. May be `lingo Macro for official language plugin (Examples)`